### ISSUE TITLE
svn export issue

### ISSUE THREAD

**Issue by yikaus** [ansible/ansible-modules-core#299]
##### Issue Type:

Bug report
##### Ansible Version:

1.7.1
##### Environment:

Ubuntu 12.04/13.10
##### Summary:

I got one issue regards to svn export  ,  file can be downloaded using svn module with export=yes , but always showing unnecessary exception and fail the task. Currently I can use ignore_errors: yes as workaround  , but still want it be reviewed .   
##### Steps To Reproduce:

run this task 

```
- name: svn export
  subversion: >
    export=yes
    repo=https://github.com/ansible/ansible-modules-core/trunk/README.md
    dest=/tmp/README.md
```
##### Expected Results:

file downloaded without exception
##### Actual Results:

file downloaded with exception as below :

```
  failed: [127.0.0.1] => {"cmd": "/usr/bin/svn --non-interactive --trust-server-cert --no-auth-cache info   /tmp/README.md", "failed": true, "rc": 1}
  stderr: svn: E155007: '/tmp/README.md' is not a working copy

 msg: svn: E155007: '/tmp/README.md' is not a working copy

 FATAL: all hosts have already failed -- aborting  
```

Thanks
Kevin

**Comment by simono** [comment]
This was fixed in fa46f3c269194bfd05b5f4b330d3e8c00d17b38d

**Comment by abadger** [comment]
In terms of releases, the fix will be in 1.9.  We tagged 1.9rc1 on Tuesday and plan on tagging 1.9rc2 on the Tuesday coming up.

With 1.9 you should be able to construct a playbook that looks like this to make things work:

```
- hosts: localhost
  tasks:
    - name: svn export
      subversion: >
        export=yes
        repo=https://github.com/ansible/ansible-modules-core/trunk/README.md
        dest=/tmp/README.md
        force=True
```

# Closing This Ticket

Hi!

We believe recent commits (likely detailed above) should resolve this question or problem for you.

This will also be included in the next major release.

If you continue seeing any problems related to this issue, or if you have any further questions, please let us know by stopping by one of the two mailing lists, as appropriate:
- https://groups.google.com/forum/#!forum/ansible-project - for user questions, tips, and tricks
- https://groups.google.com/forum/#!forum/ansible-devel - for strategy, future planning, and questions about writing code

Because this project is very active, we're unlikely to see comments made on closed tickets, but the mailing list is a great way to ask questions, or post if you don't think this particular
issue is resolved.

Thank you!
